src/main/java/exercise39/SortingRecords.java

```diff
@@ -20,6 +21,7 @@
     }
 
     public List<Map<String, String>> recordsCreator() {
+        //location?
```

Comment:
So I think this data setup should probably live in the test rather than the production code, as it's specific to our particular use-case.